pkg/autoscaler/autoscaler/util.go

```diff
@@ -146,3 +147,54 @@ func genMetricsEndpoint(tac *v1alpha1.TidbClusterAutoScaler) (string, error) {
 	}
 	return fmt.Sprintf("http://%s-prometheus.%s.svc:9090", tac.Spec.Monitor.Name, tac.Spec.Monitor.Namespace), nil
 }
+
+func autoscalerToStrategy(tac *v1alpha1.TidbClusterAutoScaler) *pdapi.Strategy {
```

Comment:
Suggest adding defaulting logic here as the code in this PR takes the assumption that all of the fields in the auto scaler CR have values.

Reply:
This is unrelated to this issue. We can implement this feature later.

Reply:
In this case, we have to review the codes in this PR later again as we're not sure if the values are defaulted correctly later.
Or we can have a separate PR to do the defaulting at the same time.

Reply:
Will implement in this PR